edx_django_utils/monitoring/internal/transactions.py

```diff
@@ -53,7 +53,9 @@ def name(self):
 
 def get_current_transaction():
     """
-    Returns the current transaction.
+    Returns the current transaction. This is only used internally and won't
```

Comment:
Can this need to be removed from`edx_django_utils/monitoring/__init__.py` and docs?